composer.json

```
@@ -14,24 +14,24 @@
         "issues": "https://github.com/acquia/http-hmac-php/issues"
     },
     "require": {
-        "php": ">=7.3.0",
+        "php": "^7.3 || ^7.4 || ^8.0 || ^8.1",
         "psr/http-message": "^1.0 || ^2.0"
     },
     "suggest": {
-        "guzzlehttp/guzzle": "^6.0",
+        "guzzlehttp/guzzle": "^6.0 || ^7.0",
         "laminas/laminas-diactoros": "^1.8 || ^2.2",
-        "symfony/security": "^4.0 || ^5.0"
+        "symfony/security": "^4.0 || ^5.0 || ^6.0"
     },
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^3.2.1",
-        "guzzlehttp/guzzle": "^6.0",
+        "guzzlehttp/guzzle": "^6.0 || ^7.0",
         "nyholm/psr7": "^1.0",
-        "php-coveralls/php-coveralls": "^2.2",
+        "php-coveralls/php-coveralls": "^2.2 || ^2.4",
```

You should use the latest version here:

```suggestion
        "php-coveralls/php-coveralls": "^2.8",
```

According to their releases (https://github.com/php-coveralls/php-coveralls/releases), they were still fixing PHP Support as of 2.6. There's no need to support multiple versions, either: the behavior of the library doesn't change based on which version of php-coveralls we use to report data.